index.html

```diff
@@ -61,7 +61,7 @@
 
   <section id="getting-started">
     <aside>
-    <h3>Alright. How do I get started?</h3>
```

Comment:
I did read it (and several others). It's not a spelling error, it is a merged word that follows the pattern of "already" and "altogether". I'm all for grammatical pedantry, and there's obviously some animosity out there toward "alright", but I don't share it.  This isn't formal writing and I have no objection. I think sidestepping the issue is probably best.